ibis/backends/snowflake/tests/test_udf.py

```diff
@@ -150,3 +154,108 @@ def cases(value, mapping):
     assert not df.empty
     assert "predicted_price" in df.columns
     assert len(df) == diamonds.count().execute()
+
+
+def touch_package(pkgpath):
+    # nix timestamps every file to the UNIX epoch for reproducibility
+    # so we modify the utime of the _copied_ code since snowflake has
+    # some annoying checks for zipping files that do not allow files
+    # older than 1980 ¯\_(ツ)_/¯
+    os.utime(pkgpath, None)
+    for root, dirs, files in os.walk(pkgpath):
+        for path in dirs + files:
+            os.utime(os.path.join(root, path), None)
+
+
+def add_packages(d, session):
```

Comment:
This is a super clever solution, much easier than zipping and staging from my previous attempts. Very nice!